<linux/limits.h> is implicitly included via <limits.h> but not required here

Source/GmmLib/Utility/GmmLog/GmmLog.cpp:36:10: fatal error: 'linux/limits.h' file not found
#include <linux/limits.h>
         ^~~~~~~~~~~~~~~~

--- Source/GmmLib/Utility/GmmLog/GmmLog.cpp.orig	2018-09-28 18:59:39 UTC
+++ Source/GmmLib/Utility/GmmLog/GmmLog.cpp
@@ -33,7 +33,6 @@ OTHER DEALINGS IN THE SOFTWARE.
 #include <sys/stat.h>
 #include <unistd.h>
 #include <fstream>
-#include <linux/limits.h>
 #endif
 
 /// Logger instance shared by all of GmmLib within a process
